waiter/src/waiter/scheduler.clj

```diff
@@ -482,9 +485,10 @@
                                         (conj scheduler-messages
                                               [:update-service-instances
                                                (assoc service-instance-info
-                                                 :service-id service-id
-                                                 :failed-instances all-failed-instances
-                                                 :scheduler-sync-time request-instances-time)])
+                                                      :failed-instances all-failed-instances
+                                                      :instance-counts {:requested instances :started task-count}
```

Comment:
Out of curiosity when have you noticed these values (`instances` and `task-count`) to be different?

Reply:
They're different in `waiter.scheduler.kubernetes`.
(I'm future-proofing. :smile:)